src/js/control/autocomplete.js

```diff
@@ -192,22 +211,78 @@ export default class controlAutocomplete extends control {
    * @param {Object} selectedOption - option - 'li' element - to be selected in autocomplete list
    */
   selectOption(list, selectedOption) {
-    const options = list.querySelectorAll('li');
-    options.forEach((option)=>{
-      option.classList.remove('active-option');
-    });
+    const options = list.querySelectorAll('li'); 
+    // --Fix for IE11
+    for (let i = 0; i < options.length; i++) {
+      options[i].classList.remove('active-option');
+    }
     if (selectedOption) {
       selectedOption.classList.add('active-option');
     }
   }
+   
 
   /**
-   * When the element is rendered into the DOM, execute the following code to initialise it
-   * @param {Object} evt - event
+   * Is the value in the autocomplete field in the pre-defined Options list?
+   * @param {Object} list - list of autocomplete options
+   * @param {Object} value -value trying to be set
+   * @return {Object} - is the option in the pre defined list
    */
+  isOptionValid(list,value){
+    const options = list.querySelectorAll('li'); 
+    let validValue = false;
+    for (let i = 0; i < options.length; i++) {
```

Comment:
minor but could probably shorten this with 
```
return options.some(options => option.innerHTML === value)
```
like loop with `break`, it will return the moment it finds match
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some

Reply:
I tried, but options does not allow the .some method.

Reply:
ah yeah youre right because its an HTMLCollection and not an array.